### PR TITLE
all: fix pre-commit-failing errors

### PR DESCRIPTION
**Pull request by liamstask**
❓ **What kind of change does this PR introduce?**
  - [x] 🐞 bug fix
  - [ ] 🐣 feature
  - [ ] 📋 docs update
  - [ ] 📋 tests/coverage improvement
  - [ ] 📋 refactoring
  - [ ] 💥 other

📋 **What is the related issue number (starting with `#`)**

n/a

❓ **What is the current behavior?** (You can also link to an open issue here)

some `pre-commit-failing` checks fail, example [here](https://github.com/cherrypy/cheroot/pull/352/checks?check_run_id=1647887998)

❓ **What is the new behavior (if this is a feature change)?**

`pre-commit-failing` check does not fail

📋 **Other information**:

📋 **Checklist**:

  - [x] I think the code is well written
  - [x] I wrote [good commit messages][1]
  - [x] I have [squashed related commits together][2] after the changes have been approved
  - [ ] Unit tests for the changes exist
  - [ ] Integration tests for the changes exist (if applicable)
  - [x] I used the same coding conventions as the rest of the project
  - [x] The new code doesn't generate linter offenses
  - [ ] Documentation reflects the changes
  - [x] The PR relates to *only* one subject with a clear title
  and description in grammatically correct, complete sentences

[1]: http://chris.beams.io/posts/git-commit/
[2]: https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit

---
This change is [Reviewable](https://reviewable.io/reviews/cherrypy/cheroot/356)

### CODE REVIEW

File: cheroot/cli.py [hunk 1]
```diff
@@ -228,7 +228,7 @@ def parse_wsgi_bind_addr(bind_addr_string):
         type=int,
         help='Timeout in seconds for putting requests into queue',
     ),
-}
+}  # noqa: C408
```
Comment (webknjaz) [inline cheroot/cli.py:231]: This violation is related to `dict()` constructor being used internally, I don't think it's related to the literal: https://github.com/adamchainz/flake8-comprehensions#c408-unnecessary-dictlisttuple-call---rewrite-as-a-literal. Also, the preference is to fix violations, not ignore them. Ignores should only be used temporarily.

**Comment by webknjaz** [comment]
@liamstask this job is non-blocking for a reason: there are extra linters that produce violations that haven't been dealt with. It's marked as "allowed to fail" so that the failing outcome wouldn't make the CI workflow red. GHA represents such jobs slightly differently in the UI than Travis CI did and I guess it's the only reason it's more annoying.

The idea is to gradually fix the violations and then migrate the linters into the main pre-commit config, once they no longer produce errors.

File: cheroot/cli.py (hunk 1 above)
Comment (liamstask) [reply cheroot/cli.py]: updated

**Review by webknjaz — COMMENTED** [review]
I'd prefer a separate atomic commit per each violation type fix. Is it still a draft?

**Comment by webknjaz** [comment]
> The idea is to gradually fix the violations and then migrate the linters into the main pre-commit config, once they no longer produce errors.

Oh, and also flakehell has a separate configuration in `pyproject.toml` but many checks are disabled. Most of them should be re-enabled before we can switch from flake8 to flakehell, that's why it's in the experimental pre-commit config.

**Comment by liamstask** [comment]
split out separate commits for each fix 👍

**Pull request merged**